contracts/mining.js

```diff
@@ -291,7 +292,7 @@ actions.createPool = async (payload) => {
   // eslint-disable-next-line no-template-curly-in-string
   const utilityTokenBalance = await api.db.findOneInTable('tokens', 'balances', { account: api.sender, symbol: "'${CONSTANTS.UTILITY_TOKEN_SYMBOL}$'" });
 
-  const authorizedCreation = api.BigNumber(poolCreationFee).lte(0)
+  const authorizedCreation = api.BigNumber(poolCreationFee).lte(0) || api.sender === api.owner
```

Comment:
Here you are skipping the balance check for the fee if api.owner is calling the contract action, but fee is still deducted further down.  Maybe api.owner should be excluded from paying the fee as it's a special case?